storymap/storage.py

```diff
@@ -15,23 +15,15 @@
 from boto.exception import S3ResponseError
 from boto.s3.connection import OrdinaryCallingFormat
 import requests
+import settings
 
 # Get settings module
-settings = sys.modules[os.environ['FLASK_SETTINGS_MODULE']]
+# settings = sys.modules[os.environ['FLASK_SETTINGS_MODULE']]
```

Comment:
after restoring this, set your local env var `FLASK_SETTINGS_MODULE=core.settings`